details_energy.md

```diff
@@ -0,0 +1,177 @@
+---
+layout: index
+title: The GCAM Water Model
+prev: supply_energy.html
+next: supply_energy.html
+gcam-version: v5.3 
+---
+
+This page provides more detailed explanations of the descriptions provided in the [Energy Supply](supply_energy.html) and [Energy Demand](demand_energy.html) modeling pages.
+
+{:toc}
+
+## Resources
+
+### Depletable Resources 
+
+The figures below show some illustrative examples of supply curves used within GCAM.
+
+<img src="gcam-figs/oil_supply_curve.png" width="750" height="300" />
+{: .fig}
+<img src="gcam-figs/gas_supply_curve.png" width="750" height="300" />
+{: .fig}
+<img src="gcam-figs/coal_supply_curve.png" width="750" height="300" />
+**Illustrative examples of supply curves and total supplies for fossil resources in GCAM **
+{: .fig}
+
+#### Representation of unconventional oil as a technology within crude oil
+
+Unconventional oil and crude oil are merged into 1 resource (crude oil). Note that the ability for countries to produce unconventional oil is maintained by adding it as a subresource and technology within crude oil. This structure allows GCAM to load all costs and energy inputs related to unconventional oil production directly as technology costs. Since unconventional oil is combined with crude oil,they share a single price that governs supply expansion.   
+
+### Renewable Resources
+
+#### Wind
+
+The supply curves in each region are derived from bottom-up analysis documented in [Zhou et al. (2012)](details_energy.html#zhou2012).
```

Comment:
I suggest:  "The supply curves in each region are derived from bottom-up analysis documented in  [Eurek et al. (2017)](details_energy.html#eurek2017)."